openapi.yaml

```diff
@@ -0,0 +1,675 @@
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id being bound
+          required: true
+          schema:
+            type: string
+        - name: binding_id
+          in: path
+          description: binding id being bound
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: parameters for the requested service binding
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ServiceBindingRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceBinding'
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceBinding'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '422':
+          description: Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      summary: deprovision of a service binding
+      tags:
+        - ServiceBindings
+      operationId: serviceBinding.unbinding
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: id of the instance associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: binding_id
+          in: path
+          description: id of the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan associated with the binding being deleted
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
```

Comment:
I'm not 100% sure, but I think `error` and `description` field should be allowed to be returned with a 400

Reply:
Tracking in #403